Neos.ContentRepositoryRegistry/Classes/Service/EventMigrationService.php

```diff
@@ -253,11 +253,11 @@ public function migrateReferencesToMultiFormat(\Closure $outputFn): void
         $this->copyEventTable($backupEventTableName);
 
         $streamName = VirtualStreamName::all();
-        $eventStream = $this->eventStore->load($streamName, EventStreamFilter::create(EventTypes::create(EventType::fromString(NodeReferencesWereSet::class))));
+        $eventStream = $this->eventStore->load($streamName, EventStreamFilter::create(EventTypes::create(EventType::fromString('NodeReferencesWereSet'))));
         foreach ($eventStream as $eventEnvelope) {
             $outputRewriteNotice = fn(string $message) => $outputFn(sprintf('%s@%s %s', $eventEnvelope->sequenceNumber->value, $eventEnvelope->event->type->value, $message));
             if ($eventEnvelope->event->type->value !== 'NodeReferencesWereSet') {
-                continue;
+                throw new \RuntimeException(sprintf('Unhandled event: %s', $eventEnvelope->event->type->value));
```

Comment:
we could _probably_ even skip the check altogether now with the filter?

Reply:
jup but when writing these kind of things i prefer to have this as sanity check :D helps sometimes ^^